Example/Tests/SerializableSpec.swift

```diff
@@ -6,7 +6,7 @@ import Faro
 
 extension Zoo: Serializable {
 
-    var json: [String : Any?] {
+    var json: [String : Any] {
```

Comment:
🎈 Nice-to-have: sommige services flippen als er keys ontbreken (doorgaans die die op Jersey gebaseerd zijn). Dan zou je nulls als values kunnen aanvaarden en er gewoon een key instoppen met als value "".